assignments/walk_tests_repo_test.go

```diff
@@ -93,3 +139,28 @@ WORKDIR /quickfeed
 		t.Errorf("readTestsRepositoryContent() mismatch (-wantAssignments +gotAssignments):\n%s", diff)
 	}
 }
+
+const invalidTypesFolder = "testdata/invalidJsonTests/invalidTypes"
```

Comment:
Should be define inside the test function where they apply unless they are needed in the global package name space.